Src/Add-AzureDtlVM.ps1

```diff
@@ -1,64 +1,142 @@
+<#
+.SYNOPSIS 
+    This script adds the specified number of Azure virtual machines in the specified DevTest Lab.
+
+.DESCRIPTION
+    It allows the creation of the VMs inside a specific lab. This script needs to be scheduled for the creation of the VMs of each lab. 
```

Comment:
Change 'This script needs to be ...' to 'This script can be run either from command line or Azure Automation' in all the rest of source files.